plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -2887,6 +2888,24 @@ else if (object.getBukkitEntity() instanceof Hanging) {
             return new ElementTag(humanEntity.isHandRaised());
         });
 
+        // <--[tag]
+        // @attribute <EntityTag.bookshelf_slot>
+        // @returns ElementTag(Number)
+        // @description
+        // Returns the Chiseled Bookshelf slot that the entity is looking at, if any.
+        // See also <@link tag LocationTag.slot>
+        // -->
+        registerSpawnedOnlyTag(ElementTag.class, "bookshelf_slot", (attribute, object) -> {

```

Comment:
This tag only works on 1.20+ (that's when `getSlot` was added it seems), so should probably be moved to be with the rest of the 1.20+ tags, inside the `if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20))` below